### PR TITLE
Upgrade default templates on launch

### PR DESCRIPTION
**Pull request by wiggin77**
#### Summary
This PR implements phase 1 of https://docs.google.com/document/d/1LHB6Kqks6kHxj1I7bJOpshiLxx-IEmtx3_tjixdbCyE/edit?usp=sharing

- selectively import default templates as embedded as Boards archive (JSONL)
    - only import if old default templates are detected
- modify board titles (remove NEW)
- remove old default templates
- do not touch user templates

Does not include image support, which will come as a separate PR and ticket.

#### Ticket Link
https://community.mattermost.com/boards/workspace/qgsck6cts3fwpqwyjiupjm5cde/b4ctmm917q3rzdfi451qtuxjzcw/150d1464-6d59-4c39-810f-ee201439dbaf/7dstzuz9yjnn3pz3ycxo3od8k9r

### CODE REVIEW

**Comment by wiggin77** [comment]
@jespino, @sbishel, this is no longer a draft.  Final default templates were added.

File: server/services/store/sqlstore/archive.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+			if err2 := s.importArchiveLine(&archiveLine, args); err2 != nil {
+				return fmt.Errorf("error importing archive line %d: %w", lineNum, err2)
+			}
+		}
+
+		if errRead != nil {
+			if errors.Is(errRead, io.EOF) {
+				break
+			}
+			return fmt.Errorf("error reading archive line %d: %w", lineNum, err)
```
Comment (wiggin77) [inline server/services/store/sqlstore/archive.go:76]:
```suggestion
			return fmt.Errorf("error reading archive line %d: %w", lineNum, errRead)
```

**Review by sbishel — APPROVED** [review]
Looks good.

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**